### ISSUE TITLE
The value passed to VARIANT() is not a valid experiment name

### ISSUE THREAD

**Issue by sryze** [ampproject/amphtml#23086]
## What's the issue?

When I load my page I see the following error:

```
The value passed to VARIANT() is not a valid experiment name:some-name
```

I did not receive this error with previous experiments that had hyphens in names. It seems like a breaking change, no?

## How do we reproduce the issue?

1. Create an experiment with hyphens/dashes in name in Google Optimize (this step is probably not required)
2. Add it to the page
3. Visit the page and open dev tools

## What browsers are affected?

* Firefox 68.0b12
* Chrome 75

## Which AMP version is affected?

1906191709480

**Comment by torch2424** [comment]
@sryze Thank you for the issue! 😄 

Could you share which version of amp-experiment you are using? 

Thank you! 👍

**Comment by sryze** [comment]
0.1

**Comment by sryze** [comment]
Found the root cause - experiment name inside `VARIANT()` did not much any experiments declared in `<amp-experiment>...</amp-experiment>` on my page, i.e. not related to `-` characters in the name.

It would be nice to have a better error message for this kind of mistake.

**Comment by torch2424** [comment]
@sryze Glad you found the issue! And thanks for letting us know. I'll go ahead and keep this issue open to change the error name.

Thank you! 😄

**Comment by micajuine-ho** [comment]
Closed by #26153
